01/10/2024 11:44:35 AM - ERROR - __main__ - 'Personagem' object has no attribute 'setNome'
Traceback (most recent call last):
  File "c:\Users\kevin\VSProjetos\AutoProducao\main.py", line 2165, in teste
    self.modificaPersonagem()
  File "c:\Users\kevin\VSProjetos\AutoProducao\main.py", line 1970, in modificaPersonagem
    personagem.setNome(novoNome)
    ^^^^^^^^^^^^^^^^^^
AttributeError: 'Personagem' object has no attribute 'setNome'. Did you mean: 'pegaNome'?
